test/e2e/operator_test.go

```diff
@@ -2100,6 +2100,46 @@ func TestLocalWithFallbackOverrideForNodePortService(t *testing.T) {
 	}
 }
 
+// TestReloadIntervalUnsupportedConfigOverride verifies that the operator
+// configures router pod replicas with the specified value for RELOAD_INTERVAL
+// if one is specified using an unsupported config override on the
+// ingresscontroller.
+func TestReloadIntervalUnsupportedConfigOverride(t *testing.T) {
+	icName := types.NamespacedName{Namespace: operatorNamespace, Name: "reload-interval"}
+	domain := icName.Name + "." + dnsConfig.Spec.BaseDomain
+	ic := newPrivateController(icName, domain)
+	if err := kclient.Create(context.TODO(), ic); err != nil {
+		t.Fatalf("failed to create ingresscontroller: %v", err)
+	}
+	defer assertIngressControllerDeleted(t, kclient, ic)
+
+	if err := waitForIngressControllerCondition(t, kclient, 5*time.Minute, icName, availableConditionsForPrivateIngressController...); err != nil {
+		t.Fatalf("failed to observe expected conditions: %w", err)
```

Comment:
```suggestion
		t.Fatalf("failed to observe expected conditions: %v", err)
```
Could this be whats causing the e2e-aws-operator job to keep failing?

Reply:
Fixed.  Thanks!